examples/fastify-server/sandbox.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "view": "console",
+  "container": {
+    "node": "14"
+  }
```

Comment:
```suggestion
```

The node 14 requirement is only really to build tRPC itself because of some silly issue with newer versions of TypeScript and Node 16 that I haven't gotten around to looking at.